README.md

```diff
@@ -12,6 +12,15 @@ Note: Unlike other silverstripe/gha-* repositories, this one is a [reusable work
 
 Create the following file in your module, and substitute the tagged version for the most recent tag prefixed with a `v` e.g. `@v1`
 
+> [!WARNING]
+> Note that the `actions: write` permission won't be used in third-party repositories, but still needs to be defined. This permission is required because in commercially supported repositories and repositories in the "silverstripe" GitHub organisation we dispatch a separate workflow which tags patch releases.
```

Comment:
Why are we mentioning the `actions:write` permission here? I thought that won't matter if it's used in a third-party repository since it's "hidden" behind an API call and only triggered it it's in the silverstripe org?

In the sample below we're only mentioning pull-requests:read and contents:read

Reply:
Oops, that should be removed.
Done.